Jenkins/Jenkinsfile

```diff
@@ -29,7 +29,7 @@ node('be-integration') {
                 isReleaseBuild = true
             } else if (env.GIT_BRANCH ==~ /\d+\.\d+\.\d+/) {
                 version = "${env.GIT_BRANCH}-${currentBuild.number}"
-                isReleaseBuild = true
+                isReleaseBuild = !env.GIT_BRANCH.startsWith("0");
```

Comment:
This allows us to push branches like 0.0.x that are debug test branches automatically built and pushed to services artifactory. It is then possible to build and deploy mbr build from that.
Question: should we also automatically build debug if the isReleaseBuild is false?